projects/smoldot-browser-demo/package.json

```diff
@@ -13,9 +13,9 @@
   ],
   "scripts": {
     "deep-clean": "yarn clean && rm -rf node_modules",
-    "clean": "rm -rf dist/ && mkdir -p dist/assets",
-    "typecheck": "tsc --noEmit",
-    "typecheck:watch": "tsc --noEmit --watch",
+    "clean": "rm -rf dist/ tsconfig.tsbuildinfo && mkdir -p dist/assets",
+    "typecheck": "tsc -b",
```

Comment:
by removing -noEmit then "typecheck" command also compiles the code instead of just typechecking. We are using parcel for that

Reply:
As above